Jenkinsfile.private

```diff
@@ -0,0 +1,30 @@
+pipeline {
+	agent { node { label 'node-08' } }
+	stages {
+		stage('Prepare workspace') {
+			steps {
+				checkout scm
+			}
+		}
+		stage('Install dependencies') {
+			steps {
+				sh 'npm install --verbose'
+			}
+		}
+		stage('Run vulnerabilities check') {
+			steps {
+				withCredentials([string(credentialsId: 'liskhq-snyk-token', variable: 'SNYK_TOKEN')]) {
+					sh '''
+					if [ -r ~/.lisk-template.package.json.sha1sum ]; then
```

Comment:
Should be `~/.lisk.package.json.sha1sum`, not `~/.lisk-template.package.json.sha1sum`